data/wsl/Autounattend_UEFI.xml

```diff
@@ -376,6 +376,22 @@
           <Order>63</Order>
           <Path>%windir%\System32\reg.exe ADD "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\PersonalizationCSP" /v DesktopImageStatus /t REG_DWORD /d 1 /f</Path>
         </RunSynchronousCommand>
+        <RunSynchronousCommand wcm:action="add">
+          <Order>64</Order>
+          <Path>powershell.exe -NoProfile -Command "Invoke-WebRequest -Uri https://github.com/grisu48/openqa-agent/releases/download/v1.1.2/agent-Windows-amd64 -OutFile 'C:\Program Files\openqa-agent.exe'"</Path>

```

Comment:
Could it be possible to have it in the data folder of the os-autoinst repo instead?

Reply:
A Windows binary? Why would we want to do that, given that pulling assets from github.com has proven to be reliable and work just fine? :thinking:

Reply:
Yes, I meant to add it to `https://github.com/os-autoinst/...` instead of `https://github.com/grisu48/...`

Reply:
`openqa-agent` is only at https://github.com/grisu48/openqa-agent for now.

Reply:
I'd copy that folder into the `data/wsl` one for having all the tools "centralized". But I guess it's a minor detail :)